### DISCUSSION TITLE
Sysex Issues

### DISCUSSION THREAD

**Discussion by lowtechlowlife** [BobanSpasic/MiniDexedLibrarian#6] [Q&A]
I recently finished my miniDexed build and I trying to use the librarian to "send bank over sysex".  I can see in my midi monitor that it is send data from the program but I am not seeing any change on the mini dexed. I have confirmed my din connection is working fine with my midi controller so I can safely rule out it being a hardware issue.  Is there a setting in the minidexed.ini file that I need to change to allow me to receive sysex information?

**Comment by BobanSpasic** [comment]
Hi lowtechlowlife,
the implementation is missing on the MiniDexed side. As for now, you can only use SD-Card to transfer banks and performances to MiniDexed.
The only thing that works at the moment, on MiniDexed side, is receiving single voices.
However, you can use the librarian to send banks to a real DX7, or to VST plugins.
Sorry, but I can't really help here. It depends on MiniDexed developers: https://github.com/probonopd/MiniDexed/issues/748

**Reply by lowtechlowlife** [reply]
Thanks for the quick reply BobanSpasic.  I thought that might be the case.  I am modifying my 3d printed case with an SD card slot now.  Link if you are interested. https://www.printables.com/model/1074872-mini-dexed-synthesizer-case

**Reply by BobanSpasic** [reply]
I have edited your post. The link was broken.

Nice case. It is probably more worth to post it to the MiniDexed discussions
